Integrations/Lockpath_KeyLight_v2/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+## [Unreleased]
```

Comment:
no need to add changelogs for new integrations.

Reply:
just remove the the file?